### ISSUE TITLE
itoadu.com fuzzy after update

### ISSUE THREAD

**Issue by CoolDino1** [ruffle-rs/ruffle#6756]
### Describe the bug

![image](https://user-images.githubusercontent.com/98194448/164023924-d6ea017d-fa70-42c6-91a4-7f26a7306685.png)
![Screenshot 2022-04-19 at 09-04-25 iTOADu com Christian Games Activities and Bible Quizzes for Kids!](https://user-images.githubusercontent.com/98194448/164024049-ddaff05d-795a-4b51-8ad6-0d33709036d7.png)
the bottom one is the april 18 version on the firefox app store(0.1.0.445), the top one is the april 19 version on the firefox app store(0.1.0.446)

### Expected behavior

I expected it to look the same (maybe a small graphics issue, the white line at the bottom, which is in another issue, fixed)

### Affected platform

Browser's extension

### Operating system

Windows 10

### Browser

Mozilla Firefox 99.0.1

**Comment by Herschel** [comment]
Thanks for the report. If you go to the extension settings and turn the log level to Info, could you press F12 copy+paste the output from the console here? Thank you!

**Comment by CoolDino1** [comment]
For some reason it is not happening on this computer, I will send the errors from the affected computer

**Comment by CoolDino1** [comment]
I have tried using another computer and I have found only some are affected, maybe it has to do with the fact that the affected ones are older?

**Comment by danielhjacobs** [comment]
> Falling back to WebGL1

Yeah, it's probably only affecting the WebGL1 backend. That fallback is usually only needed for supporting older computers. What caused that backend to stop working properly is a separate question.
